README.md

```diff
@@ -73,6 +73,7 @@ import visearch from 'visearch-javascript-sdk';
 // Set up keys
 visearch.set('app_key', 'YOUR_APP_KEY');
 visearch.set('tracker_code', 'YOUR_TRACKER_CODE');
+visearch.set('placement_id', 'YOUR_PLACEMENT_ID'); // required for products based API
```

Comment:
Product Search would have a different endpoint e.g. `search.visenze.com` instead of `visearch.visenze.com`. For Product Search, the tracker code will be set to app key and placement ID without needing user to input. User did not need to specify this.

Reply:
But the tracker code is still required for the current visearch API right?

Reply:
yes it is required for current API. What I mean is to update the README to indicate that there is no need to set the tracker code for products search APIs.